core/src/main/java/com/gentics/mesh/core/endpoint/node/BinaryFieldResponseHandler.java

```diff
@@ -130,22 +129,29 @@ private void resizeAndRespond(RoutingContext rc, BinaryGraphField binaryField, I
 				imageParams.setFocalPoint(fp);
 			}
 		}
+		Integer originalHeight = binaryField.getBinary().getImageHeight();
+		Integer originalWidth = binaryField.getBinary().getImageWidth();
+
+		if (imageParams.getHeight().equals("auto")) imageParams.setHeight(originalHeight);
+		if (imageParams.getWidth().equals("auto")) imageParams.setWidth(originalWidth);
```

Comment:
Please always use braces for `if`
https://stackoverflow.com/questions/2125066/is-it-a-bad-practice-to-use-an-if-statement-without-curly-braces